$OpenBSD: patch-src_fw_paravirt_h,v 1.4 2021/09/01 11:04:21 sthen Exp $

Allow detection of >4GB RAM. Normally seabios only allows this with the
QEMU config device which VMM does not have.

Index: src/fw/paravirt.h
--- src/fw/paravirt.h.orig
+++ src/fw/paravirt.h
@@ -75,4 +75,5 @@ int qemu_cfg_write_file(void *src, struct romfile_s *f
 int qemu_cfg_write_file_simple(void *src, u16 key, u32 offset, u32 len);
 u16 qemu_get_romfile_key(struct romfile_s *file);
 
+void vmm_check_high_mem(void);
 #endif
